flexget/components/sites/sites/site_rutracker.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+
+from loguru import logger
+from requests.exceptions import RequestException
+
+from flexget import plugin
+from flexget.components.sites.urlrewriting import UrlRewritingError
+from flexget.event import event
+
+from urllib.parse import urlparse, parse_qs
+
+
+logger = logger.bind(name='rutracker')
+
+
+class SiteRutracker:
+    schema = {'type': 'boolean'}
+
+    base_url = 'https://api.t-ru.org'
+
+    # urlrewriter API
+    def url_rewritable(self, task, entry):
+        url = entry['url']
+        if url.startswith('https://rutracker.org/forum/viewtopic.php?t='):
+            return True
+        return False
+
+    @plugin.internet(logger)
+    def url_rewrite(self, task, entry):
+        """
+            Gets torrent information for topic from rutracker api
+        """
+
+        url = entry['url']
+        logger.info('rewriting download url: %s' % url)
```

Comment:
please use curly braces for loggers:
```suggestion
        logger.info('rewriting download url: {}', url)
```